src/VisualStudio.Common/GlobalSuppressionsOptionsPageControl.xaml.cs

```diff
@@ -20,39 +19,73 @@ public GlobalSuppressionsOptionsPageControl()
 
         private void OpenLocation_Click(object sender, RoutedEventArgs e)
         {
-            string assemblyPath = typeof(GlobalSuppressionsOptionsPageControl).Assembly.Location;
+            string appDataFolderPath = Path.Combine(
+                Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),
+                @"JosefPihrt\Roslynator\VisualStudio\2019");
 
-            if (!string.IsNullOrEmpty(assemblyPath))
-            {
-                string assemblyDirPath = Path.GetDirectoryName(assemblyPath);
+            string ruleSetPath = Path.Combine(appDataFolderPath, "roslynator.ruleset");
 
-                if (!string.IsNullOrEmpty(assemblyDirPath))
+            if (!File.Exists(ruleSetPath))
+            {
+                try
                 {
-                    string ruleSetPath = Path.Combine(assemblyDirPath, "roslynator.ruleset");
+                    string defaultRuleSetFileName = GetDefaultRulesetFileName();
+                    if (File.Exists(defaultRuleSetFileName))
+                    {
+                        Directory.CreateDirectory(appDataFolderPath);
 
-                    try
+                        File.Copy(defaultRuleSetFileName, ruleSetPath);
+                    }
+                }
+                catch (Exception ex)
+                {
+                    if (ex is IOException || ex is UnauthorizedAccessException)
+                    {
+                        MessageBox.Show(ex.Message, null, MessageBoxButton.OK, MessageBoxImage.Error);
+                    }
+                    else
                     {
-                        string fileToSelect = (File.Exists(ruleSetPath)) ? ruleSetPath : assemblyPath;
+                        throw;
+                    }
+                }
+            }
 
-                        Process.Start("explorer.exe", $"/select, \"{fileToSelect}\"");
+            if (File.Exists(ruleSetPath))
+            {
+                try
+                {
+                    Process.Start("explorer.exe", $"/select, \"{ruleSetPath}\"");
+                }
+                catch (Exception ex)
+                {
+                    if (ex is InvalidOperationException
```

Comment:
You could use exception filters here, to make debugging easier.
See https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-6#exception-filters and https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/when.